pages/osx/apachectl.md

```diff
@@ -0,0 +1,15 @@
+# apachectl
+
+> Apache HTTP Server control interface for macOS.
+
+- Load the org.apache.httpd launchd job:
```

Comment:
I am a bit curious about why the words "Load" and "Unload" are used and not "start" and "stop".

Reply:
That's from the official `man` page but it's true that it doesn't fit the needs of this repo.

Reply:
Yes, we try to avoid copying text verbatim from the man page. And try to change it to something simple and clear.